addon/modifiers/did-intersect.js

```diff
@@ -1,44 +1,56 @@
 import Modifier from 'ember-modifier';
+import { inject as service } from '@ember/service';
+import { assert } from '@ember/debug';
 
 export default class DidIntersectModifier extends Modifier {
-  // Public API
-  handler = null;
-  options = {};
+  @service('ember-scroll-modifiers@observer-manager') observerManager;
 
-  // Private API
-  observer = null;
+  _isObservable = true;
 
-  observe() {
-    if ('IntersectionObserver' in window) {
-      this.observer = new IntersectionObserver((entries, observer) => {
-        this.handler(entries[0], observer);
-      }, this.options);
+  constructor() {
+    super(...arguments);
+    this._isObservable = 'IntersectionObserver' in window;
```

Comment:
**Question:** is it worth logging something here, maybe with `console.assert` or `console.info`? I'm thinking about when the scenario would come up, and it's not obvious to me—is it just 🔥 DIAF IE11 🔥 or are there other browsers where it wouldn't run? Whatever we decide, it doesn't need to block this PR, just thinking about how consumers would figure out/test/etc. if this was just “mysteriously” not working in a given context.

Reply:
This exists solely to prevent it from completely borking IE. A comment next to where `_isObservable` is defined on the class that explains this might be helpful, particularly as a reminder to remove it when we want to drop support for IE11 altogether.

Reply:
Yeah, not-borking-IE seems good; I like the approach of commenting by the declaration of `_isObservable` as the solution here.